.github/workflows/try.yml

```diff
@@ -22,8 +22,8 @@ jobs:
                   body
                 })
             }
-
-            let tokens = "${{ github.event.comment.body }}".split(" ");
+            let comment = context.payload.comment; 
+            let tokens = comment.body.split(" ");
```

Comment:
We should split on `/+s/` instead of `" "` here as well in order to properly handle newlines in the comment.